examples/pytorch/BertNewsClassification/README.md

```diff
@@ -1,22 +1,5 @@
-## Bert news classification example with MLFlow
-
-In this example, we have used a pretrained Bert model to classify the news reports into any one of the 4 different categories("world", "Sports", "Business", "Sci/Tech").
-The autolog code uses Pytorch Lightning's MLFlowLogger to log metrics.
-The code is trained using pytorch lightning loop and the autolog function call in the main - `mlflow.pytorch.autolog()`
-is responsible for logging the params, metrics, model summary and the model.
- 
-
-### Code related to MLflow:
-* [`mlflow.pytorch.autolog`]
-This is an experimental api that logs ML model artifacts and metrics.
-The metrics are logged during training of the model.
-
-## Setting tracking URI
-MLflow tracking URI can be set using the environment variable MLFLOW_TRACKING_URI
-
-Example: `export MLFLOW_TRACKING_URI=http://localhost:5000/`
-
-For more details - https://mlflow.org/docs/latest/tracking.html#where-runs-are-recorded
+## BERT news classification example
```

Comment:
@karthik-77 I reworked this file with a couple of changes:
* Simplified the intro & removed the "code related to MLflow" section, which is redundant given the description of ``mlflow.pytorch.autolog`` here
* Split out the ``mlflow ui`` command into its own "Viewing results in the UI" section and moved it above the description of how to customize training parameters (to streamline the common path of running the example with default parameters -> viewing results in the UI).
* Reworded the description of training parameters 

Would you be able to apply similar changes to the other READMEs? Thanks!

Reply:
I have applied similar changes in the other read me files (Mnist-example1,example2) and updated the PR